samples/videoDecodePerf/videodecodeperf.cpp

```diff
@@ -37,11 +37,12 @@ THE SOFTWARE.
 #include "roc_video_dec.h"
 #include "common.h"
 
-void DecProc(RocVideoDecoder *p_dec, VideoDemuxer *demuxer, int *pn_frame, double *pn_fps) {
+void DecProc(RocVideoDecoder *p_dec, VideoDemuxer *demuxer, int *pn_frame, double *pn_fps, int thread_id) {
     int n_video_bytes = 0, n_frame_returned = 0, n_frame = 0;
     uint8_t *p_video = nullptr;
     int64_t pts = 0;
     double total_dec_time = 0.0;
+    p_dec->SetDecoderSessionID(thread_id);
```

Comment:
Instead of using an integer as the thread_id here, use the actual thread_id by calling `std::this_thread::get_id()`. This way, you don't need to pass `int thread_id` to `DecProc` function.